examples/decoding/plot_haxby_anova_svm.py

```diff
@@ -46,7 +46,7 @@
 
 ### Prediction function #######################################################
 
-### Define the prediction function to be used.
+# Define the prediction function to be used.
```

Comment:
Removing them sounds fine then. Note that if we want headers for sphinx-gallery we'll have to write them in reST.